### PR TITLE
pr-VTOL_controller_diagram

### PR DESCRIPTION
**Pull request by sfuhrer**
This PR adds a VTOL control overview to Concepts/ PX4 Architectural Overview/Controller Diagrams.

### CODE REVIEW

File: en/flight_stack/controller_diagrams.md [hunk 1]
```diff
@@ -39,6 +39,27 @@ The roll and pitch controllers have the same structure and the longitudinal and
...
+
+## VTOL Flight Controller 
+
+![VTOL Attitude Controller Diagram](../../assets/diagrams/VTOL_controller_diagram.png)
+
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1tVpmFhLosYjAtVI46lfZkxBz_vTNi8VH/view?usp=sharing
+Request access from dev team. -->
+
+This section gives a short overview on the control structure of Vertical Take-off and Landing (VTOL) aircraft. 
+The VTOL flight controller consists of both the multicopter and fixed-wing controllers, either seperatly in the corresponding flight modes, or together during transitions.
```
Comment (hamishwillee) [inline en/flight_stack/controller_diagrams.md]: Typo - seperatly

File: en/flight_stack/controller_diagrams.md [hunk 2]
```diff
@@ -39,6 +39,27 @@ The roll and pitch controllers have the same structure and the longitudinal and
...
+
+![VTOL Attitude Controller Diagram](../../assets/diagrams/VTOL_controller_diagram.png)
+
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1tVpmFhLosYjAtVI46lfZkxBz_vTNi8VH/view?usp=sharing
+Request access from dev team. -->
+
+This section gives a short overview on the control structure of Vertical Take-off and Landing (VTOL) aircraft. 
+The VTOL flight controller consists of both the multicopter and fixed-wing controllers, either seperatly in the corresponding flight modes, or together during transitions. 
+The diagram above presents a simplified control diagram from the attitude setpoint $$\Psi_{sp}$$ to the actuator setpoints. 
+For a standard and tilt-motor VTOL (link to airframes), during transition the fixed-wing attitude controller produces the rates setpoint, which are then fed into the seperate rate controllers, resulting in torque commands for the multicopter and fixed-wing actuators.
```
Comment (hamishwillee) [inline en/flight_stack/controller_diagrams.md]: Feels like this might perhaps be a new paragraph.

**Comment by hamishwillee** [comment]
Looks good. I'll see if we can get a technical review when you move it out of draft.

**Comment by sfuhrer** [comment]
@RomanBapst have a look

File: en/flight_stack/controller_diagrams.md [hunk 3]
```diff
@@ -39,6 +39,31 @@ The roll and pitch controllers have the same structure and the longitudinal and
...
+![VTOL Attitude Controller Diagram](../../assets/diagrams/VTOL_controller_diagram.png)
+
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1tVpmFhLosYjAtVI46lfZkxBz_vTNi8VH/view?usp=sharing
+Request access from dev team. -->
+
+This section gives a short overview on the control structure of Vertical Take-off and Landing (VTOL) aircraft. 
+The VTOL flight controller consists of both the multicopter and fixed-wing controllers, either running separately in the corresponding VTOL modes, or together during transitions. 
+The diagram above presents a simplified control diagram. 
+Note the VTOL attitude controller block, which mainly facilitates the necessary switching and blending logic for the different VTOL modes, as well as VTOL-type-specific control actions during transitions (e.g. ramping up the pusher motor of a standard VTOL during forward transition). 
+The inputs into this block are called "virtual" as depending on the current VTOL mode, some are ignored by the controller.
```
Comment (hamishwillee) [inline en/flight_stack/controller_diagrams.md]: Comma after "as" in "as depending"

File: en/flight_stack/controller_diagrams.md [hunk 4]
```diff
@@ -39,6 +39,31 @@ The roll and pitch controllers have the same structure and the longitudinal and
...
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1tVpmFhLosYjAtVI46lfZkxBz_vTNi8VH/view?usp=sharing
+Request access from dev team. -->
+
+This section gives a short overview on the control structure of Vertical Take-off and Landing (VTOL) aircraft. 
+The VTOL flight controller consists of both the multicopter and fixed-wing controllers, either running separately in the corresponding VTOL modes, or together during transitions. 
+The diagram above presents a simplified control diagram. 
+Note the VTOL attitude controller block, which mainly facilitates the necessary switching and blending logic for the different VTOL modes, as well as VTOL-type-specific control actions during transitions (e.g. ramping up the pusher motor of a standard VTOL during forward transition). 
+The inputs into this block are called "virtual" as depending on the current VTOL mode, some are ignored by the controller. 
+
+For a standard and tilt-rotor VTOL, during transition the fixed-wing attitude controller produces the rates setpoint, which are then fed into the separate rate controllers, resulting in torque commands for the multicopter and fixed-wing actuators.
```
Comment (hamishwillee) [inline en/flight_stack/controller_diagrams.md]: I think it is "rate setpoints" not "rates setpoint" here.

File: en/flight_stack/controller_diagrams.md [hunk 5]
```diff
@@ -39,6 +39,31 @@ The roll and pitch controllers have the same structure and the longitudinal and
...
+For tailsitters, during transition the multicopter attitude controller is running. 
+
+The outputs of the VTOL attitude block are separate torque and force commands for the multicopter (actuator_control_0) and fixed-wing (actutor_controls_1) actuators.
```
Comment (hamishwillee) [inline en/flight_stack/controller_diagrams.md]: Typo actutor_controls_1

YOu might want to put these in backticks - ie `actuator_control_0` to make them more visible.

**Comment by sfuhrer** [comment]
@roman have a look

**Comment by sfuhrer** [comment]
@hamishwillee Do you think we could merge this? Or any work necessary from our side?

**Comment by hamishwillee** [comment]
@sfuhrer I thought this was good. I was just waiting on Roman to review in response to your request..... but then I just noticed you tagged the wrong Roman!
Anyway, merged now, and thanks.

**Comment by hamishwillee** [comment]
@sfuhrer PS, please do ping me if something is stuck that you think can move on. I would eventually have followed this up, but not until I ran low on urgent priority tasks ... maybe in a few months.

**Comment by sfuhrer** [comment]
> @sfuhrer I thought this was good. I was just waiting on Roman to review in response to your request..... but then I just noticed you tagged the wrong Roman!
> Anyway, merged now, and thanks.

uups yeah that's the wrong roman^^


> @sfuhrer PS, please do ping me if something is stuck that you think can move on. I would eventually have followed this up, but not until I ran low on urgent priority tasks ... maybe in a few months.

Totally understand, I pinged you as soon as I remembered that this was still open. 

Thanks!

**Pull request merged**